src/main/java/org/sefglobal/scholarx/service/MailConnection.java

```diff
@@ -0,0 +1,27 @@
+package org.sefglobal.scholarx.service;
+
+import org.simplejavamail.api.mailer.Mailer;
+import org.simplejavamail.mailer.MailerBuilder;
+import org.springframework.core.env.Environment;
+
+public class MailConnection {
+    private static Environment env;
+
+    private MailConnection(Environment env){
+        this.env = env;
+    }
+    private static String username = env.getProperty("spring.mail.username");
```

Comment:
The environment variables are not working for some reason. Could find the reason yet

Reply:
we can refer this as well https://www.simplejavamail.org/configuration.html#section-config-properties

Reply:
we can refer this as well https://www.simplejavamail.org/configuration.html#section-config-properties

Reply:
I think it's better if we can add those properties as custom properties. What we currently doing is kind of a hack, right?

Reply:
We can just add those in a file named `simplejavamail.properties` file (according to this). It automatically gets the properties from the file.

I tried it with a `simplejavamail.yml` file and it also works!

These are the steps I followed:
1. Create a new file as `simplejavamail.yml` in the `src/main/resources` directory.
```
simplejavamail:
  smtp:
    host: smtp.gmail.com
    port: 587
    username: ${EMAIL}
    password: ${APP_PASSWORD}
  defaults:
    from:
      name: Sustainable Education Foundation
      address: ${EMAIL}
```

2. Remove the Environment related things we previously added to `MailConnection.java` and `EmailUtil`.
3. We can remove the `from()` field from the email object (in `EmailUtil.java`) because the EmailBuilder takes it from the properties as the default value. 
4. In `MailConnection.java`, we can remove the `.withSMTPServer(...)` part because it also gets the default values from properties.
```
    public static Mailer getInstance() {
        if (instance == null){
            instance = MailerBuilder.buildMailer();
        }
        return instance;
    }
```
5. Remove the mail-related properties from the `application.yml` file 

cc: @pasindur99

Reply:
> I tried it with a simplejavamail.yml file and it also works!

Update: The `.yml` format doesn't work. Let's just use the `.properties` format.

simplejavamail.properties:

```
simplejavamail.smtp.host=smtp.gmail.com
simplejavamail.smtp.port=587
simplejavamail.smtp.username=${EMAIL}
simplejavamail.smtp.password=${APP_PASSWORD}
simplejavamail.defaults.from.name=Sustainable Education Foundation
simplejavamail.defaults.from.address=${EMAIL}
```

Reply:
Fixed with [7e6e220](https://github.com/sef-global/scholarx/pull/345/commits/7e6e220bc231560d65cc4877cf37228144719d38)
@Piumal1999  Can you have a look, please?